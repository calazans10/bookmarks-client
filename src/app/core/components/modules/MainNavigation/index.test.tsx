import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import { MainNavigation } from '.';

describe('MainNavigation', () => {
  it('renders without crashing', () => {
    const { container } = render(
      <MainNavigation pathname="/home" title="Home" onRequestLogout={jest.fn()} />,
      { wrapper: MemoryRouter }
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
