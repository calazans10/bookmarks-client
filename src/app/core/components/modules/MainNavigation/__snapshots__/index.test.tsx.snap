// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNavigation renders without crashing 1`] = `
<nav
  class="sc-AxjAm llwbHT"
>
  <ul>
    <li
      class="sc-AxirZ bIIUiU"
    >
      <a
        href="/home"
      >
        Home
      </a>
    </li>
  </ul>
</nav>
`;
